src/main/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActor.scala

```diff
@@ -91,6 +92,8 @@ private class TaskLauncherActor(
     localRegion: () => Option[Region]) extends Actor with StrictLogging with Stash {
   // scalastyle:on parameter.number
 
+  import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
why not `context.dispatcher`?

Reply:
I copied form other actors but it's a good point. Do we want to use the same thread pool as the actor system?